### PR TITLE
fx(DB/Quest): The Stories Dead Men Tell Quest Fix

### PR DESCRIPTION
**Pull request by Vanheden**
Added script to the graves.
Fixed with WoWDatabaseEditor

## Changes Proposed:
Add so it plays the scenes etc.

## Issues Addressed:
Fixed the Roleplay at the Graves.
https://www.youtube.com/watch?v=GZKXSFzAhhs&ab_channel=Roddan
Closes: https://github.com/azerothcore/azerothcore-wotlk/issues/16885

## SOURCE:
Source from Sniff & Youtube: https://www.youtube.com/watch?v=FOctMDsnJDM&ab_channel=WorldOfWarcraftGuidesAndGameplay

## Tests Performed:
I tested it ingame several times and it works.

## How to Test the Changes:

1. .q add 13643
2. Click the graves.

## Known Issues and TODO List:
It plays a death sound when clicking on the graves when the NPC spawns, on videos they don't.

### CODE REVIEW

**Comment by elthehablo** [comment]
Works well. The npcs die when they are spawned, this is the cause of the sound and the animation. Afaik there is a way to change them to have a permanent "dead" state. Probably have to change that in `creature_template_addon`

**Comment by elthehablo** [comment]
image

**Comment by Vanheden** [comment]
> Works well. The npcs die when they are spawned, this is the cause of the sound and the animation. Afaik there is a way to change them to have a permanent "dead" state. Probably have to change that in `creature_template_addon`

Yeah I used Sniff data to find out the spells etc. On the first grave **Sir Wendell Balfour** I had to add his Feign Death as a Aura instead of spellcast. For some reason he didn't spawn dead like **Lorien Sunblaze** did.

**Comment by elthehablo** [comment]
buttt if it's sniffed (which you did a nice job as, as far as I can tell) I really wonder how blizz did it

**Comment by Vanheden** [comment]
> buttt if it's sniffed (which you did a nice job as, as far as I can tell) I really wonder how blizz did it

Yeah also noticed the Dwarf when he's dead. He lays on his stomach but from a video 6 months ago he lays on his back and from a video back in legion he lays dead on his stomach to, kinda weird

**Comment by elthehablo** [comment]
> > buttt if it's sniffed (which you did a nice job as, as far as I can tell) I really wonder how blizz did it
> 
> Yeah also noticed the **Conall Irongrip** when he's dead. He lays on his stomach but from a video 6 months ago he lays on his back and from a video back in legion he lays dead on his stomach to, kinda weird

maybe difference between models/animations? idk

**Comment by Vanheden** [comment]
> > > buttt if it's sniffed (which you did a nice job as, as far as I can tell) I really wonder how blizz did it
> > 
> > 
> > Yeah also noticed the **Conall Irongrip** when he's dead. He lays on his stomach but from a video 6 months ago he lays on his back and from a video back in legion he lays dead on his stomach to, kinda weird
> 
> maybe difference between models/animations? idk

Yeah can be.

File: data/sql/updates/pending_db_world/questfix.sql [hunk 1]
```diff
@@ -0,0 +1,67 @@
+
+ -- Sir Wendell's Grave smart ai
+SET @ENTRY := 194537;
+DELETE FROM `smart_scripts` WHERE `source_type` = 1 AND `entryOrGuid` = @ENTRY;
+UPDATE `gameobject_template` SET `AIName` = 'SmartGameObjectAI' WHERE `entry` = @ENTRY;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
+(@ENTRY, 1, 0, 0, 71, 0, 100, 0, 21077, 0, 0, 0, 12, 33439, 3, 19000, 0, 0, 0, 8, 0, 0, 0, 8461.727, 468.7472, 596.2335, 4.729842, 'On game object event 21077 triggered - Self: Summon creature Sir Wendell Balfour (33439) at (8461.727, 468.7472, 596.2335, 4.729842) as summon type timed despawn with duration 19 seconds');
+
+
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 22 AND `SourceEntry` = 194537 AND `SourceId` = 1;
```
Comment (Maria-sequel) [inline data/sql/updates/pending_db_world/questfix.sql:10]: you can combine these with the other sourceId= 1 with 
```AND `SourceEntry` IN ( 194537, 194539, etc) ```
Same with comibining the sourceid = 0's.

Also, you can keep 1 whiteline between different sets, but it's common to use only 1 whiteline on AC

**Review by Maria-sequel — CHANGES_REQUESTED** [review]

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**